lib/xml_builder.ex

```diff
@@ -25,11 +25,11 @@ defmodule XmlBuilder do
   end
 
   defmacrop is_blank_list(list) do
-    quote do: is_nil(unquote(list)) or (is_list(unquote(list)) and unquote(list) == [])
+    quote do: is_nil(unquote(list)) or unquote(list) == []
   end
 
   defmacrop is_blank_map(map) do
-    quote do: is_nil(unquote(map)) or (is_map(unquote(map)) and map_size(unquote(map)) == 0)
+    quote do: is_nil(unquote(map)) or unquote(map) == unquote(Macro.escape(%{}))
```

Comment:
Could this be shortened to:

```suggestion
    quote do: is_nil(unquote(map)) or unquote(map) == %{}
```

Reply:
Seems like we can! I thought maps always had to be escaped when used in macros, but turns out that's not true 👍